CacheMoney/backend/pom.xml

```diff
@@ -37,6 +37,11 @@
 			<artifactId>java-jwt</artifactId>
 			<version>3.18.3</version>
 		</dependency>
+		<dependency>
+            <groupId>com.microsoft.sqlserver</groupId>
+            <artifactId>mssql-jdbc</artifactId>
+            <version>10.2.0.jre11</version>
+        </dependency>
```

Comment:
If you look further down you will notice that mssql-jdbc is already listed as a dependency, yours is probably from setup before it was put on main, but we only need one.